velox/exec/tests/TableWriteTest.cpp

```diff
@@ -1985,6 +1988,49 @@ TEST_P(PartitionedWithoutBucketTableWriterTest, fromSinglePartitionToMultiple) {
       "SELECT * FROM tmp");
 }
 
+TEST_P(PartitionedTableWriterTest, removePartitionColumns) {
```

Comment:
Do we need a new test or could we modify existing tests to verify the list of columns written into the file? I assume there are existing tests that write partitioned tables. These test would start failing once we add verification for list of columns in the file, no?

Reply:
@mbasmanova  Removed.